publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
awesome, btw if u have some time in ur hand, check https://www.youtube.com/watch?v=UxFgkCUiRAI i think it could give us some ideas about routing, comm with db, custom templating, etc...